packages/fiori-elements-writer/test/alp.test.ts

```diff
@@ -2,14 +2,21 @@ import type { FioriElementsApp } from '../src';
 import { generate, TemplateType, LROPSettings } from '../src';
 import { join } from 'path';
 import { removeSync } from 'fs-extra';
-import { testOutputDir, debug, getTestData, feBaseConfig, projectChecks } from './common';
+import {
+    testOutputDir,
+    debug,
+    getTestData,
+    feBaseConfig,
+    projectChecks,
+    updatePackageJSONDependencyToUseLocalPath
+} from './common';
 import type { OdataService } from '@sap-ux/odata-service-writer';
 import { OdataVersion } from '@sap-ux/odata-service-writer';
 import type { ALPSettings, ALPSettingsV2, ALPSettingsV4 } from '../src/types';
 import { TableSelectionMode, TableType, WorklistSettings } from '../src/types';
 
 const TEST_NAME = 'alpTemplates';
-jest.setTimeout(120000); // Needed when debug.debugFull
+jest.setTimeout(360000); // Needed when debug.debugFull
```

Comment:
That looks just wrong (not just your change but the line as it was before already.
If this is really only required for debugging, could we please only call `jest.setTimeout` when actually debugging? Maybe use the `UX_DEBUG` env variable that we use for other debugging activities

Reply:
Addressed in [86a8810](https://github.com/SAP/open-ux-tools/pull/984/commits/86a8810a9d8ec735f42f5700c60d1f9d460d1569)